apps/epic-react/src/components/lesson-completion-toggle.tsx

```diff
@@ -84,15 +84,19 @@ const LessonCompleteToggle = ({className}: {className?: string}) => {
             flattenedLessons &&
             flattenedLessons[currentLessonIndex + 1]
           ) {
-            reward()
```

Comment:
getting rid of these reward animations as we are triggering this optimistically and this double renders the animation